packages/helium-cli/src/bootstrap.ts

```diff
@@ -1,481 +0,0 @@
-import {
```

Comment:
Why's this file being deleted?

Reply:
Replaced with create-dao and create-subdao